presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -59,6 +59,46 @@ General Aggregate Functions
 
     Returns the geometric mean of all input values.
 
+.. function:: grouping(col1, ..., colN) -> integer
+
+    Returns a bit set converted to decimal, indicating which columns are present in a grouping. The function
+    must be used in conjunction with ``GROUPING SETS``, ``ROLLUP`` or ``CUBE`` and its arguments must
+    match exactly the columns referenced in the corresponding ``GROUPING SETS``, ``ROLLUP`` or ``CUBE``.
+
+    To compute the resulting bit set for a particular row, bits are assigned to the argument columns with
+    the rightmost column being the least significant bit. For a given grouping, a bit is set to 0 if the
```

Comment:
You're right, that's a more accurate description. Would you like me to rephrase the docs and Java doc in those terms? Your suggestion certainly makes sense to me but I've spent a lot of time thinking about grouping. I think explaining the function in terms of columns included in the group by will make more sense to a new user. For example, that's how Postgres explains the function: `Bits are assigned with the rightmost argument being the least-significant bit; each bit is 0 if the corresponding expression is included in the grouping criteria of the grouping set generating the result row, and 1 if it is not.` (bottom of page: https://www.postgresql.org/docs/9.6/static/functions-aggregate.html)